### PR TITLE
adding docker file

### PR DESCRIPTION
**Pull request by keis**
cleaned up my dockerfile from klarnasque stuff.

### CODE REVIEW

File: misc/eremetic.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "id": "/eremetic",
+  "cpus": 0.2,
+  "mem": 100.0,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "alde/eremetic",
```
Comment (keis) [inline misc/eremetic.json]: Not sure if you have this account. if not it should be changed :smile_cat:

Comment (alde) [inline misc/eremetic.json]: I don't, but maybe I should :)

Comment (alde) [inline misc/eremetic.json]: now I do

**Comment by keis** [comment]
d'oh looks like travis found the makefile :anguished:

**Comment by alde** [comment]
lgtm

**Pull request merged**